src/components/monitor/PwnedStatusSection.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Breach, HaveIbeenPwnedText, Paste } from '@/assets/types/have-i-been-pawned';
+import { PwnedSection } from './PwnedSection';
+import { AllGoodSection } from './AllGoodSection';
+
+interface PwnedStatusSectionProps {
+  breaches: Breach[];
+  resultPastes: Paste[];
+  textContent: {
+    PwnedSection: HaveIbeenPwnedText['HeroSection']['PwnedSection'];
+    AllGoodSection: HaveIbeenPwnedText['HeroSection']['AllGoodSection'];
+  };
+}
+
+const PwnedStatusSection: React.FC<PwnedStatusSectionProps> = ({ breaches, resultPastes, textContent }) => {
+  return breaches.length > 0 ? (
```

Comment:
Maybe you can extract this check in a constant, like: `const isEmailPwned = breaches.length > 0` or something like that, so when other teammate reads the code, it is more readable and easy to understand, don't you think?

Reply:
Perfect, updated.